nixos/modules/misc/nixpkgs.nix

```diff
@@ -130,6 +130,18 @@ in
       '';
     };
 
+    crossSystem = mkOption {
+      type = types.nullOr types.attrs;
```

Comment:
Don't let me forget about this when I return to https://github.com/NixOS/nixpkgs/pull/34444

Reply:
Can we have a similar value for localSystem? O:).

Reply:
Yes if `system` before is new in 18.03, it should be changed to local system instead.

Reply:
I'd be happy to do this in a follow-up PR though.